apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/lib/seller-celebration-modal/use-has-seen-seller-celebration-modal.js

```diff
@@ -0,0 +1,31 @@
+import apiFetch from '@wordpress/api-fetch';
+import { useState, useEffect } from '@wordpress/element';
+
+const useHasSeenSellerCelebrationModal = () => {
+	const [ hasSeenSellerCelebrationModal, setHasSeenSellerCelebrationModal ] = useState( '' );
+
+	useEffect( () => {
+		fetchHasSeenSellerCelebrationModal();
+	} );
+
+	function fetchHasSeenSellerCelebrationModal() {
+		apiFetch( { path: '/wpcom/v2/block-editor/has-seen-seller-celebration-modal' } )
+			.then( ( result ) =>
+				setHasSeenSellerCelebrationModal( result.has_seen_seller_celebration_modal )
+			)
+			.catch( () => setHasSeenSellerCelebrationModal( false ) );
+	}
+
+	function updateHasSeenSellerCelebrationModal( value ) {
+		apiFetch( {
+			method: 'PUT',
+			path: '/wpcom/v2/block-editor/has-seen-seller-celebration-modal',
+			data: { has_seen_seller_celebration_modal: value },
```

Comment:
Do we need to do any kind of validation/sanitization on `value` ?

Reply:
I don't think so, since we validate the argument to the endpoint as being a boolean in the API endpoint.